docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+group: guide
+title: Setup and Installation Guide
+---
+
+* TOC
+{:toc}
+
+
+<span class="c29">Yavin UI browser support</span>
+-------------------------------------------------
+
+<span class="c11">Yavin UI is an HTML-based browser application that is
+supported on the most recent three major versions of the following
+browsers:</span>
+
+<span class="c11"></span>
+
+-   <span class="c11">Chrome</span>
+-   <span class="c11">Firefox</span>
+-   <span class="c11">Safari</span>
+
+<span class="c29">Yavin installation guide</span>
+-------------------------------------------------
+
+The “<span class="c40">Installation Guide</span><span class="c11">” link
+takes you through the step-by-step guide in installing all the packages
+and files needed to have a full operational Yavin on your system.
+</span>
+
+<span class="c29">About data ingestion and data sources</span>
+--------------------------------------------------------------
+
+Before you can use Yavin to visualize and analyze data, you need to
+configure the Elide portion of Yavin to access the data you wish to
+report on. You do so by defining a <span class="c48">data
+source</span><span class="c11"> including the database connection
+configuration itself and its schema.</span>
+
+### <span class="c9">Data source dialects</span>
+
+<span class="c11">Elide analytic APIs generate SQL queries against your
+target database(s). Elide must be configured with a Dialect to correctly
+generate native SQL that matches the database grammar. Elide supports
+the following dialects by default:</span>
+
+
+|      Name                       |  Availiability                        |
+|---------------------------------|---------------------------------------|
+| <span class="c40">H2</span>     | <span class="c22">Now</span>          |
+| <span class="c40">Hive</span>   | <span class="c22">Now</span>          |
+| <span class="c40">Presto</span> | <span class="c22">Now</span>          |
+| <span class="c33">MySQL</span>  | <span class="c22">Now</span>          |
+{:.table}
+
+More information on dialects and how to use them can be found at: <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23dialects&amp;sa=D&amp;ust=1606228594025000&amp;usg=AOvVaw3kCKuvbnXUCJCPe8SCcGF7" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#dialects</a></span><span
+class="c11"> </span>
+
+<span class="c29">About data models</span>
+------------------------------------------
+
+A semantic model is a simplified representation of the data that resides
+in your database.  Semantic models consist of concepts like Tables,
+Dimensions, and Measures.  While the physical database schema (The
+schema describes the data, how it is organized and managed in the
+physical data source) can be complex, the semantic model is flat (there
+are no relationships between tables) and the columns are referenced by
+simple names (as opposed to complex formulas or derivations). A <span
```

Comment:
I'm not sure about this paragraph starting "A data model is a...".  A few points:
 - We have two terms - semantic model and data model.  We need to be consistent.
 - I'm not sure what you mean about data being loaded.  Loading data is orthogonal to Yavin.
 - Isolating models on a per service basis - what multiple services?
 - What do you mean by design that emphasizes isolation?

I don't understand this paragraph.  I think it is confusing and introducing concepts like multiple services and isolation that really aren't necessary to understand how to use yavin.